### PR TITLE
Infra encapsulation: backwards compatible changes

### PR DESCRIPTION
**Pull request by lluiscampos**
Collecting in this PR some changes I am making in the test framework that are backwards compatible. This will ease the final review.

The main change is the imports clean-up

### CODE REVIEW

**Comment by lluiscampos** [comment]
This can be reviewed already. The tests passed (once) and that is all the evidence we need, as any error in the import would have make a test to constantly fail.

FYI @oleorhagen ^

File: tests/pytest.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Disable stout capture, as it conflicts with Fabric
```
Comment (oleorhagen) [inline tests/pytest.ini]: s/stout/stdout/

**Comment by lluiscampos** [comment]
> How about running this on it also? https://pypi.org/project/isort/
> Just to verify.

I didn't know about this tool. Thanks! At the moment it is a half way reorder and I will tackle it again in the refactor.

File: tests/tests/common_update.py [hunk 2]
```diff
@@ -13,15 +13,17 @@
...
 
-from ..common_docker import *
+import tempfile
+import random
+
+from fabric.api import *
+import pytest
+
+from .. import conftest
 from ..common import *
```
Comment (mchalski) [inline tests/tests/common_update.py]: left on purpose?

Comment (mchalski) [reply tests/tests/common_update.py]: I mean I see the commit message, but why everything else and not this?

Comment (lluiscampos) [reply tests/tests/common_update.py]: Yes, see commit comment: https://github.com/mendersoftware/integration/pull/742/commits/9b6aba7acf92911208a0d4a7ae9fc5f3e8f22af1

Basically the common is overriding some fabric functions, so it must be this way (for now, at least).

File: tests/pytest.ini (hunk 1 above)
Comment (lluiscampos) [reply tests/pytest.ini]: :eyes:

**Comment by lluiscampos** [comment]
Tests passed here: [Pipeline 95988319](https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/95988319). The last force-push is a typo + rebase from master.

Merging

File: tests/tests/common_update.py (hunk 2 above)
Comment (lluiscampos) [reply tests/tests/common_update.py]: I should have been explicit in the commit message, you are right!

**Pull request merged**